sklearn/datasets/tests/test_svmlight_format.py

```diff
@@ -257,17 +263,21 @@ def test_dump():
                     assert_array_equal(X2.sorted_indices().indices, X2.indices)
 
                     X2_dense = X2.toarray()
+                    if sp.issparse(X_input):
```

Comment:
```suggestion
                    X_input_dense = X_input.toarray() if sp.issparse(X_input) else X_input
```

Reply:
Tried that, but it would be over 80 char long..

Reply:
OK pitty. So  good to be merged.